repose-aggregator/components/filters/herp/src/main/scala/org/openrepose/filters/herp/HerpFilter.scala

```diff
@@ -54,6 +54,8 @@ class HerpFilter @Inject()(configurationService: ConfigurationService,
   private final val DEFAULT_CONFIG = "highly-efficient-record-processor.cfg.xml"
   private final val X_PROJECT_ID = "X-Project-ID"
   private final val X_METHOD_LABEL: String = "X-METHOD-LABEL"
+  private final val X_PP_USER: String = "X-PP-User"
+  private final val X_PP_GROUPS: String = "X-PP-Groups"
```

Comment:
Considering this is a `final val` (i.e., a constant) of an obvious type (i.e., `String`), I would say that annotating the type is not really necessary. That's just personal preference, though.